modules/ducktests/tests/ignitetest/services/utils/sql_util.py

```diff
@@ -0,0 +1,48 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This module contains JDBC driver wrapper.
+"""
+
+import random
+import jaydebeapi
+from ignitetest.services.ignite import IgniteService
+from ignitetest.utils.version import DEV_BRANCH, IgniteVersion
+
+
+def connection(ignite_service: IgniteService, ver: IgniteVersion = None):
+    """
+    :param ignite_service: IgniteService.
+    :param ver: IgniteVersion jdbc driver for connection.
+    :return Connection.
+    """
+    if ver is None:
+        ver = ignite_service.config.version
+
+    vstr = ver.vstring
```

Comment:
why it is better than use str(ver)?

Reply:
str(DEV_BRANCH) = 'dev'
DEV_BRANCH.vstring = '2.10.0'
we use ignite-core-2.10.0-SNAPSHOT.jar

Reply:
Do we use '__version__ = '2.10.0.dev0' somewhere now?
The better case is to detect dev version automatically (see ignitetest.services.utils.ignite_spec.ApacheIgniteApplicationSpec.__jackson for details) and memoize it (see ignitetest.services.utils.decorators.memoize)